packages/renderer/src/lib/ui/EmptyScreen.svelte

```diff
@@ -6,6 +6,7 @@ import { onMount } from 'svelte';
 export let icon: any;
 export let title = 'No title';
 export let message = 'Message';
+export let detail = undefined;
```

Comment:
need to init to empty string probably so we know it's a string type

Reply:
Done.